.github/workflows/ci.yml

```diff
@@ -7,7 +7,24 @@ on:
     branches: [ main ]
 
 jobs:
-  build-and-test:
+  lint:
```

Comment:
Don't spin up a bot for a tiny job.

Reply:
I think its fine, since e.g. when there is a linting error, ESLint would annotate it for every bot on the files multiple times. In order to avoid that we run it once.